lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -395,7 +395,6 @@ $data-table-cell-min-width-file-size: $data-table-cell-min-width !default;
         min-height: inherit;
         align-items: center;
         word-break: break-all;
-        width: 100%;
```

Comment:
this affects every table cell, and should be thoroughly tested

Reply:
I could maybe limit it to cells with `adf-datatable-cell--image` as a class to limit it a bit more, however I'd imagine this could still have an impact